marketplace/deployer_util/clean_iam_resources.sh

```diff
@@ -19,12 +19,11 @@ set -eox pipefail
 [[ -z "$NAME" ]] && echo "NAME must be set" && exit 1
 [[ -z "$NAMESPACE" ]] && echo "NAMESPACE must be set" && exit 1
 
-# Delete the service account and RBAC objects by labels.
+# Delete the service account (which owns its RBAC objects) by label.
```

Comment:
Does it mean that only RBAC objects owned by the service account will be deleted?

Reply:
Yes.
That is the intention of this `kubectl delete` call - to delete the deployer's RBAC objects (hence the label), which will now be owned by the deployer service account.

Added a comment to point to the kinds list in case other types need to be deleted later.

Reply:
(Note that this script is for deleting resources associated with the deployer right after deployment completes, not for deleting application resources when the app is deleted. Other application resources are still deleted with the app, as they are owned by the Application)